services/QuillLMS/lib/tasks/queries.thor

```diff
@@ -43,6 +43,26 @@ class Queries < Thor
     end
   end
 
+  # user_id 9874030 is a test user
+  # e.g. bundle exec thor queries:print_snapshot_sql 9874030 'sentences-written'
+  desc 'print_snapshot_sql user_id query_key', 'Print the SQL and data usage for a snaphot query to the console'
```

Comment:
Really like the idea of having this tool and building it into our regular processes to track performance and improvements.